docs/howto/how-to-version-range-upgrades.md

```diff
@@ -5,7 +5,7 @@ Set the version for the desired package in the Catalog source to a comparison st
 Example:
 
 ```yaml
-apiVersion: olm.operatorframework.io/v1
+apiVersion: olm.operatorframework.io/v1alpha1
```

Comment:
This actually should be v1, not v1alpha1. We completed the v1 API bump for the ClusterExtension API as of #1228

Reply:
Right.  I was going to push changes to that PR, but it ... merged under me, and I mistakenly transplanted the v1alpha1 stuff.